pkg/network/ebpf/c/protocols/http2/decoding-defs.h

```diff
@@ -168,6 +168,7 @@ typedef struct {
 typedef struct {
     __u64 response_last_seen;
     __u64 request_started;
+    __u64 tags;
```

Comment:
I think we can use `__u8` here
we're trying to keep the struct as thin as possible, as it can results in adding multiple of MBs to the WSS of the agent

Reply:
I've changed this now. HTTP uses `__u64`, perhaps that also should be changed later.

Reply:
indeed
you can create a follow up PR with HTTP change as well